services/hosted_services.md

```diff
@@ -1,19 +1,91 @@
 # Hosted Systems in Use at PUL
 
-## tigris.princeton.edu
+## Infolinx/Tigris
 
 ### Purpose
-Records management software used by both University staff and Library staff in the Library's Records Management. Anne-Marie Phillips is the primary contact for the software. The software product name for this is "Infolinx". Further information available at https://aisww.com/infolinx/. 
+Records management software used by both University staff and Library staff in the Library's Records Management. Anne-Marie Phillips is the primary contact for the software. The software product name for this is "Infolinx". [Further information](https://aisww.com/infolinx/). 
 
 ### Access
-Visit https://tigris.princeton.edu. Tigris is integrated with our campus IDP. Access to it is governed by membership in certain Active Directory groups established specifically for tigris. These need to be managed via https://tools.princeton.edu. 
+Web [tigris login](https://tigris.princeton.edu). Tigris is integrated with our campus IDP. Access to it is governed by membership in certain Active Directory groups established specifically for tigris. These need to be managed via OIT's [web tools portal](https://tools.princeton.edu). 
 
 Groups defined
-* TIGRIS Administrator 1 User (Anne Marie)
+* TIGRIS Administrator
 * TIGRIS Legal User
 * TIGRIS Read Write
 * TIGRIS Storage User
 * TIGRIS User  
 
 ### Renewing the SSL Cert
 Tigris.princeton.edu has an A DNS record pointing to: princetonuniversity.cloudapp.net. When the certificate is expiring a new one needs to be emailed to support for tigris at: support@gimmal.com. Must be submitted as a .pfx file. 
+
+### Getting Tigris Support
+General application support questions should be sent via the vendor [web portal](https://aisww.com/support/). 
+
+## Aeon
+
+### Purpose 
+ Aeon is used to manage requests from users to utilize our Special Collections materials our restricted access reading rooms. Patrons place requests directly in the Illiad web interface the service provides. The Library Catalog and Pulfalight also directly route requests in the Aeon web interface.. Review recent [releases](https://support.atlas-sys.com/hc/en-us/articles/360011818834-Aeon-Release-Schedule).
```

Comment:
I'm not entirely sure I understand how Aeon and ILLiad connect - would this change be correct?
```suggestion
Aeon is used to manage requests from users to utilize our Special Collections materials in our restricted access reading rooms. Patron requests to the Aeon web interface may come from the ILLiad web interface the service provides, from the Library Catalog, or from Pulfalight. Review recent [releases](https://support.atlas-sys.com/hc/en-us/articles/360011818834-Aeon-Release-Schedule).
```

Reply:
I re-wrote this section, please give it another look when you have a chance.